src/redux/features/modules/modulesAsyncThunks.ts

```diff
@@ -67,7 +67,7 @@ export const requestListModuleDeliverables = createAsyncThunk(
   async (request: { moduleId: number; projectId: number }) => {
     const searchParams: SearchRequestPayload = {
       prefix: 'projects.projectDeliverables',
-      fields: ['id', 'module_id', 'project_id', 'name', 'category', 'dueDate', 'status', 'type'],
+      fields: ['id', 'module_id', 'project_id', 'name', 'category', 'dueDate', 'status', 'type', 'type(raw)'],
       search: {
         operation: 'and',
         children: [
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 104]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->If you do go with the renaming of variables to match types: 

add above the line:
```
typeName: result.type,
type: result['type(raw)'],
```

below this line: (to remove the remapped fields
```
'type(raw)': undefined
```
<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2748?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
I revert changes to `ModuleDeliverable` type and modified `ModuleDeliverableSearchResult` type instead, then updated the `.map` to explicitly map each field. I think these changes incorporate most of your feedback and cleans things up a bit, but please let me know if this looks ok to you.